packages/office-addin-unit-test/test/officeJSMock.test.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as assert from "assert";
+import * as mocha from "mocha";
+import { OfficeJSMock } from "../src/main";
+
+const testObject = {
+  "range": {
+    "color": "blue",
+    "getColor": function() {
+      return this.color;
+    },
+    "font": {
+      "size": 12
+    }
+  },
+}
+
+describe("OfficeJSMock", function() {
```

Comment:
Could we get a better description.

Reply:
Are you thinking of anything? It kinda says the name of the object being tested and I felt it was fine.

Reply:
It's just the name of the object and doesn't give any context about the testing being done.

Reply:
Added a more descriptive name